cluster/src/dunit/scala/io/snappydata/cluster/PreparedQueryRoutingDUnitTest.scala

```diff
@@ -908,4 +908,36 @@ class PreparedQueryRoutingDUnitTest(val s: String)
     rs1.close()
     ps3.close()
   }
+
+  def testPreparedStatementUnicodeBug(): Unit = {
+    serverHostPort = AvailablePortHelper.getRandomAvailableTCPPort
+    vm2.invoke(classOf[ClusterManagerTestBase], "startNetServer", serverHostPort)
+    // scalastyle:off println
+    println(s"testPreparedStatementUnicodeBug: network server started at $serverHostPort")
+    // scalastyle:on println
+    val conn = DriverManager.getConnection(
+      "jdbc:snappydata://localhost:" + serverHostPort)
+
+    // scalastyle:off println
+    println(s"testPreparedStatementUnicodeBug: Connected to $serverHostPort")
+    val stmt1 = conn.createStatement()
+    // scalastyle:on println
+
+    stmt1.execute("create table region (val string, description string) using column")
+    stmt1.execute("insert into region values ('粤' , 'unicode')")
```

Comment:
can you change to use unicode equivalent '\u...' here so that unicode escape parsing will also be tested